diff --git a/cpu/x86/Makefile.x86 b/cpu/x86/Makefile.x86
index 0aececd..e1a0ed8 100755
--- a/cpu/x86/Makefile.x86
+++ b/cpu/x86/Makefile.x86
@@ -3,12 +3,12 @@ CONTIKI_CPU_DIRS = .
 CONTIKI_SOURCEFILES += mtarch.c elfloader-x86.c
 
 ### Compiler definitions
-CC       = gcc
-LD       = gcc
-AS       = as
-OBJCOPY  = objcopy
-STRIP    = strip
-CFLAGSNO = -Wall -g -I/usr/local/include
+CC       = aarch64-poky-linux-gcc
+LD       = aarch64-poky-linux-gcc
+AS       = aarch64-poky-linux-as
+OBJCOPY  = aarch64-poky-linux-objcopy
+STRIP    = aarch64-poky-linux-strip
+CFLAGSNO = -Wall -g -I usr/include
 CFLAGS  += $(CFLAGSNO)
 ifeq ($(HOST_OS),Linux)
   LDFLAGS  = -Wl,-Map=contiki-$(TARGET).map,-export-dynamic
