diff --git a/examples/6lbr/Makefile b/examples/6lbr/Makefile
index 7c1b2f8..a90ca3a 100755
--- a/examples/6lbr/Makefile
+++ b/examples/6lbr/Makefile
@@ -124,8 +124,8 @@ MODULES += core/net/ip64 core/net/ip64-addr
 PROJECT_SOURCEFILES += ip64-eth-driver.c
 endif
 
-tools:
-	cd tools && $(MAKE)
+#tools:
+#	cd tools && $(MAKE)
 
 6lbr-demo:
 	pushd test; ../package/usr/bin/6lbr conf-templates/cooja-1-router.conf; popd
