diff --git a/platform/cooja/Makefile.cooja b/platform/cooja/Makefile.cooja
index c28dac6..fc43177 100755
--- a/platform/cooja/Makefile.cooja
+++ b/platform/cooja/Makefile.cooja
@@ -71,7 +71,7 @@ CONTIKI_CPU=$(CONTIKI)/cpu/x86
 
 ### Compiler arguments
 #CC = gcc
-CFLAGSNO = $(EXTRA_CC_ARGS) -Wall -g -I/usr/local/include -DCLASSNAME=$(CLASSNAME)
+CFLAGSNO = $(EXTRA_CC_ARGS) -Wall -g -I usr/include -DCLASSNAME=$(CLASSNAME)
 CFLAGS   += $(CFLAGSNO)
 
 MODULES += core/net core/net/mac \
